recipes/qt-advanced-docking-system/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx]
+  - '10.12'                # [osx]
```

Comment:
Typically the SDK is increased first, then the deployment target is increase.

Also this selector should be `[osx and x86_64]` to ensure it doesn't get picked up by osx + arm

Reply:
Will these changes be ok?
```yaml
MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
  - '10.12'                # [osx and x86_64]
MACOSX_SDK_VERSION:        # [osx and x86_64]
  - '10.14'                # [osx and x86_64]
```

Reply:
yes. i think they will do.